postgres/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG - postgres
 
+* [Added] Adds option to tag metrics with replication_role. See [#2929](https://github.com/DataDog/integrations-core/pull/2929)
```

Comment:
Hey, can you revert any changes to this file? The workflow should be that this file isn't modified during PRs, then when we release the check, this will be autogenerated based on the PR title. 

(Removing these modifications will also fix up the merging conflicts 🙂)